### DISCUSSION TITLE
Meeting Notes for 2020-02-19 and 2020-02-24

### DISCUSSION THREAD

**Discussion by YairHalberstadt** [dotnet/csharplang#3278] [General]
@agocke added a couple of meeting notes yesterday. I thought I'd link them here:

https://github.com/dotnet/csharplang/blob/master/meetings/2020/LDM-2020-02-19.md
https://github.com/dotnet/csharplang/blob/master/meetings/2020/LDM-2020-02-24.md

**Comment by TonyValenti** [comment]
As it relates to equality, I think that each member in a class should be able to provide an optional equality comparer via an attribute that is used to compare that member.  This would make it very easy to “override” the default comparison on a per member basis.
